CalculatorLibrary/CalculatorLibrary.cs

```diff
@@ -25,7 +25,7 @@ public class ExpressionsHistory
         }
 
         public List<ExpressionsHistory> History { get; set; } = new();
-        public int CountExpressions { get; set; } = 0;
+        public int CountExpressions { get; set; };
```

Comment:
```suggestion
        public int CountExpressions { get; set; }
```

The semicolon prevents the project from building.